pageserver/src/tenant/storage_layer.rs

```diff
@@ -722,6 +722,7 @@ pub(crate) enum LayerId {
 struct LayerToVisitId {
     layer_id: LayerId,
     lsn_floor: Lsn,
+    lsn_ceil: Lsn,
```

Comment:
doc comment on the struct needs updating / mentioning

Reply:
https://github.com/neondatabase/neon/pull/11494/commits/a66f0648e17ee348e40464b3a65ffa6df0a40c31